README.md

```diff
@@ -1,2 +1,110 @@
-# idm-styleguide
-IDM's living styleguide
+# idm-web-uikit
+
+IDM's living web ui kit.
+
+## Base technology
+
+- React
+- CSS modules
+- [PostCSS](https://github.com/postcss/postcss) with [MOXY's preset](https://github.com/moxystudio/postcss-preset-moxy)
+- SVG spriting support with [external-svg-sprite-loader](https://github.com/karify/external-svg-sprite-loader)
+
+
+## Commands
+
+### start
+
+```sh
+$ npm run storybook
+```
+
+Starts [Storybook](https://storybook.js.org/).
+
+### build
+
+```sh
+$ npm run build
+```
+
+Builds the project.
+
+NOTE: at the current stage of the package, the build script has not yet been defined. It will be defined in the near future.
+
+### lint
+
+```sh
+$ npm run lint
+```
+
+Checks if the project has any linting errors.
+
+### release
+
+```sh
+$ npm run release
+```
+
+Releases and publishes the package. Runs tests, lints and builds the project beforehand.
+
+This command uses [`standard-version`](https://github.com/conventional-changelog/standard-version) underneath. The version is automatically inferred from the [conventional commits](https://conventionalcommits.org/).
+
+NOTE: this package still does not have a release script, as it is still in the initial stages of its development. This script will be added in the near future.
```

Comment:
One question: why are we saying this? It seems the tooling is there and the `release` script exists.

Reply:
Also in regards to this, please apply the same changes of this PR in this repo: https://github.com/ipfs-shipyard/discussify-styleguide/pull/11

Reply:
Ok, @DominguesGM is taking care of the changes.

Reply:
You're right, it doesn't make sense to say this as we do in fact have a release script. This paragraph will be removed.